workLoggingSystem.html

```diff
@@ -265,7 +265,7 @@
                         </div>
                         <div class="col">
                             <div class="row">
-                                <div class="col-md-4 menu">Difficulty:</div>
+                                <div class="col-md-4 menu" id="ddd">Difficulty:</div>
```

Comment:
No need to give them id, You can directly change the text color of input tag id's to red, will reduce the complexity of the code